roles/ceph-ansible/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# list of devices which will be used for Ceph.
+# If it is empty, list is autodetected by ceph-ansible
+devices: []
+
+# use collocated journals? Default to true for simplicity.
+collocation: true
+
+# if collocation is false and journal devices are set,
+# create journals on journal devices.
```

Comment:
What if `collocation` is `false` but `journal_devices` are not set? Does it use some _autodetection_, same as for empty `devices`, or it is not supported configuration?